src/OnlineSales/Controllers/BaseControllerWithImport.cs

```diff
@@ -55,6 +56,11 @@ public virtual async Task<ActionResult<ImportResult>> Import([FromBody] List<TI>
         {
             var importRecord = importRecords[i];
 
+            if (!AdditionalImportCheck(importRecords, i, result))
```

Comment:
@IgorKostyukov here (inside the AdditionalImportCheck method) you make a separate DB select query for every importRecord. This will lead to a lot of DB calls and slow execution of the import if you try to import lots of comments. It should be converted to batch query.